src/commands/publish.ts

```diff
@@ -70,7 +70,7 @@ export default class Publish extends Command {
     if (!config.platforms) this.error('no platforms configured');
 
     for (const [key, value] of Object.entries(config.platforms)) {
-      if (!value.executable) this.error(`No executable path found for platform ${key}`)
+      if (!value.executable && key !== "webgl") this.error(`No executable path found for platform ${key}`)
```

Comment:
we need to be able to support web as well in the platform keys in the hyperplay.yml file

i.e. a hyperplay.yml file like 

```
account: game_dev_account
project: game
release: 0.1.0

description: Release notes go here.

platforms:
  webgl: 
    path: dist/webgl/
    zip: false
  web:
    external_url: https://hyperplay.xyz
  windows_amd64: 
    path: dist/windows/amd64/
    zip: true
    executable: Game\\Game.exe
```

Reply:
might make sense to add an override variable in the platform config as well like `ignoreExecutableCheck` for a platform so this check is opt-out and a general solution

Reply:
Re: my below message, I think `ignoreExecutableCheck` is a good to have but if it's not expected for a web or a webgl build, I don't believe the user should have to add it.  I worry that the config is already getting a bit verbose, would a user have to do?

```
  webgl: 
    path: dist/webgl
    ignoreExecutableCheck: false
    isFolderUpload: false
```

Also as another thought, it may be bad to enable certain things as general solutions, for all of the native fields we want to enforce the executable, and it would actually cause a lot of problems for them to skip the check.